AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -664,7 +664,14 @@ - (ASCellNode *)dataController:(ASDataController *)dataController nodeAtIndexPat
 
 - (ASSizeRange)dataController:(ASDataController *)dataController constrainedSizeForNodeAtIndexPath:(NSIndexPath *)indexPath
 {
-  ASSizeRange constrainedSize;
+  ASSizeRange constrainedSize = [_layoutInspector collectionView:self constrainedSizeForNodeAtIndexPath:indexPath];
```

Comment:
It is possible for _layoutInspector to be nil, right?  Quite subtly, it is actually not safe to message nil for non-scalar return types.  Though it nearly always returns a properly-zero'd struct in the simulator, it is prone to returning garbage data in some fields when run on the device.  This is one of the most common causes of device-only crashers and other weird behaviors that can't be reproduced in the simulator.